testing/ui-tests/src/correct/wasm_runtime_metadata.rs

```diff
@@ -0,0 +1,14 @@
+#[subxt::subxt(wasm_file_path = "../../../../artifacts/westend_runtime.wasm")]
```

Comment:
Nit: could we call it `runtime_path` perhaps? Then we have `runtime_metadata_path` to point to the path of some runtime metadata file, and `runtime_path` to point to the path of some runtime :)

Reply:
or "runtime_wasm_path" but the native runtime is substrate is removed now? So, runtime_path is maybe sufficient :)

Reply:
I'd be happy with `runtime_wasm_path` too; either that or `runtime_path` feels consistent with the other attrs to me :)